src/styles/images/index.md.njk

```diff
@@ -8,53 +8,70 @@ layout: layout-pane.njk
 
 {% from "_example.njk" import example %}
 
-Avoid unnecessary decoration. Only use images if there’s a real user need.
+Avoid using images for unnecessary decoration. Only use images if there’s a real user need.
 
-## Aspect ratio
+This guidance is for government teams that build online services. To learn how to use images in GOV.UK mainstream content, go to [GOV.UK content guidance for images](https://www.gov.uk/guidance/content-design/images).
 
-### Photography
+## Using images to give information
 
-The aspect ratio for photographs should be 3:2.
+Services usually work best without relying on images. Focus on writing clear, simple content for your service first.
 
-{{ example({group: "styles", item: "images", example: "default", html: false, open: false, size: "l"}) }}
+Adding images can sometimes help users better understand instructions. However, images should only be visual aids — written content must contain all the information the user needs to complete the service.
```

Comment:
**Note on use of 'written content'**
GOV.UK image guidance uses 'body text' to refer to this, but I feel confident with us sticking to 'written content' as it covers the possibility of other areas of text like descriptions, captions, hint text, etc.